segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentStoreMetrics.java

```diff
@@ -528,4 +529,20 @@ public static void outstandingEventProcessorBytes(String processorName, int cont
     }
 
     //endregion
+
+    //region ContainerKeyIndex
+
+    /**
+     * Reports the number of credits used for a given Hash-Based Table Segment. Note that these Segments are exclusively
+     * used for internal metadata storage and may have a special name format. For this reason, we report the Segment name
```

Comment:
Are we going to see full name starting from the underscore .."_system/<stream>/<segmnt>" with this?

Reply:
Yes, that's correct. If you see the first screenshot I posted, we also see the segment name like `_system/containers/storage_metadata_0` (using the `segmentTags` method). But other tags are added apart from that one, I think. Due to the different naming that these internal segments may have (also in tests), I opted to just report the credits and the segment name, so we are sure that parsing the tags do not lead into issues.